docs/oxAuth_swagger.yaml

```diff
@@ -0,0 +1,3382 @@
+                               to identify the RP session at the OP when the logout_uri is used. If omitted, the default value is false.
+                backchannel_logout_uri:
+                  type: string
+                  description: RP URL that will cause the RP to log itself out when sent a Logout Token by the OP.
+                backchannel_logout_session_required:
+                  type: boolean
+                  description: Boolean value specifying whether the RP requires that a session ID Claim be included in the Logout Token 
+                          to identify the RP session with the OP when the backchannel_logout_uri is used. If omitted, the default value is false.  
+                request_uris:
+                  type: array
+                  description: Provide a list of request_uri values that are pre-registered by the Client for use at the Authorization Server.
+                  items:
+                    type: string 
+                scopes: 
+                  type: string
+                  deprecated: true
+                  description: This param will be removed in a future version because the correct is 'scope' not 'scopes', see (rfc7591).
+                claims:
+                  type: string
+                  description: String containing a space-separated list of claims that can be requested individually.                  
+                id_token_token_binding_cnf:
+                  type: string
+                  description: Specifies the JWT Confirmation Method member name (e.g. tbh) that the Relying Party expects
+                        when receiving Token Bound ID Tokens. The presence of this parameter indicates that the Relying Party 
+                        supports Token Binding of ID Tokens.
+                        If omitted, the default is that the Relying Party does not support Token Binding of ID Tokens.                  
+                tls_client_auth_subject_dn:
+                  type: string
+                  description: An string representation of the expected subject distinguished name of the certificate, 
+                             which the OAuth client will use in mutual TLS authentication.                  
+                allow_spontaneous_scopes:
+                  type: boolean
+                  description: Specifies whether to allow spontaneous scopes for client. The default value is false.
+                  example: false      
+                spontaneous_scopes:
+                  type: array
+                  description: List of spontaneous scopes 
+                  items:
+                    type: string
+                run_introspection_script_before_access_token_as_jwt_creation_and_include_claims:
+                  type: boolean
+                  description: Boolean value with default value false. If true and access_token_as_jwt=true then run introspection script 
+                              and 
```

Comment:
token_type_hint is not required

Reply:
The screenshot seems mismtach...
I have removed required attribute for token_type_hint from 
![image](https://user-images.githubusercontent.com/43700552/76413340-3fe18580-63bb-11ea-921b-973dfec2a4ed.png)

Reply:
When I reviewed it I think it wasn't mistmached. Maybe you updated PR and it shifted?